packages/react-button/src/components/SplitButton/stories/SplitButtonDefault.stories.tsx

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
-/* eslint-disable @typescript-eslint/ban-ts-comment */
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
 import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { SplitButton } from './SplitButton';
+/* eslint-enable @typescript-eslint/ban-ts-comment */
```

Comment:
If you are using `next-line` above this line isn't needed anymore
```suggestion
```

Reply:
yeah, i did a bunch of find and replace. might have added some extras by mistake